client-v2/src/shared/components/article/Article.tsx

```diff
@@ -163,7 +161,6 @@ class ArticleComponent extends React.Component<ComponentProps> {
               isUneditable={!!article && isUneditable}
               onDelete={onDelete}
               onAddToClipboard={onAddToClipboard}
-              notifications={article ? notifications : []}
```

Comment:
Just double checking that this doesn't leave us with unwanted renders? There are quite a few literals around and I presume getArticleData isn't memoized so probably not

Reply:
This is a removed line, unless I've missed your point?!

Reply:
Yep, I'm a moron! 🥇

Reply:
I won't have you say that, chiefly because I've done this more than once and _I_ am not a moron 😎